projects/addon-doc/src/tokens/page-loaded.ts

```diff
@@ -0,0 +1,8 @@
+import {InjectionToken} from '@angular/core';
+import {defer, Observable, of, timer} from 'rxjs';
+import {switchMapTo} from 'rxjs/operators';
+
+export const TUI_DOC_PAGE_LOADED = new InjectionToken<Observable<boolean>>(
+    'Stream that emits if loading of page is over (for example, to begin scrollIntoView)',
```

Comment:
`[TUI_DOC_PAGE_LOADED] Stream that emits...`

Reply:
fixed